docs/modules/plugins/pages/plugin-browser-stack.adoc

```diff
@@ -78,6 +78,44 @@ When I save BrowserStack network logs to $scopes variable `$variableName`
 . `$scopes` - xref:parameters:variable-scope.adoc[variable scopes]
 . `$variableName` - name of variable to save network logs
 
+See <<_enable_network_capturing>>
+
+.Save network logs into variable
+[source,gherkin]
+----
+When I close mobile application
+When I save BrowserStack network logs to SCENARIO variable `networkLogs`
+Then number of JSON elements from `${networkLogs}` by JSON path `$..*[?(@.url == 'https://graph.example.com/v8.0')]` is equal to 1
+----
+
+=== Save network logs into context
+
+Saves network traffic captured during application run into JSON context. The network traffic data is in JSON format. The application session must be closed before network logs can be saved.
+
+[source,gherkin]
+----
+When I save BrowserStack network logs to JSON context
+----
+
+See <<_enable_network_capturing>>
+
+The followinig steps can be used to interact with the JSON context:
+
+* xref:plugins:plugin-rest-api.adoc#_verify_context_contains_data[Verify context contains data]
+* xref:plugins:plugin-rest-api.adoc#_verify_number_of_elements_in_context[Verify number of elements in context]
+* xref:plugins:plugin-rest-api.adoc#_save_element_from_context[Save element from context]
+* xref:plugins:plugin-rest-api.adoc#_save_number_of_elements_in_context[Save number of elements in context]
```

Comment:
let's just put the link to JSON steps section, no one will update this section while adding new JSOn steps